tests/helper.py

```diff
@@ -0,0 +1,17 @@
+import sys
```

Comment:
Is this class used somewhere? Maybe it can be moved to conftest file.

Reply:
Nah, accidental push